sbin/signalhandler.sh

```diff
@@ -1,6 +1,27 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# For terminal colors
+esc=$(echo -en "\033")
```

Comment:
Yep, I look forward to the contribution ;-)  I took the lazy way out of fixing the undefined vars, but reuse would be better.